src/main/resources/application.yml

```diff
@@ -16,7 +16,7 @@ chain:
   id: ${IEXEC_BLOCKCHAIN_ADAPTER_API_CHAIN_ID:65535}
   node-address: ${IEXEC_BLOCKCHAIN_ADAPTER_API_NODE_ADDRESS:http://localhost:8545} #https://viviani.iex.ec/
   block-time: ${IEXEC_BLOCKCHAIN_ADAPTER_API_BLOCK_TIME:1} #in seconds, use 15 for mainnet
-  hub-address: ${IEXEC_BLOCKCHAIN_ADAPTER_API_HUB_ADDRESS:0xBF6B2B07e47326B7c8bfCb4A5460bef9f0Fd2002} #0x3eca1B216A7DF1C7689aEb259fFB83ADFB894E7f
+  hub-address: ${IEXEC_BLOCKCHAIN_ADAPTER_API_HUB_ADDRESS:0xC129e7917b7c7DeDfAa5Fff1FB18d5D7050fE8ca} #0x3eca1B216A7DF1C7689aEb259fFB83ADFB894E7f
```

Comment:
1. Leave conf as is was and only update itest values
or
2. Update default `block-time` & `is-sidechain` in this hotfix as well